lib/discover/global.go

```diff
@@ -448,3 +449,15 @@ func (c *contextClient) Post(ctx context.Context, url, ctype string, data io.Rea
 	req.Header.Set("Content-Type", ctype)
 	return c.Client.Do(req)
 }
+
+func globalDiscoveryIdentity(addr string) string {
+	return "global discovery server " + addr
+}
+
+func ipv4DiscoveryIdentity(port int) string {
+	return fmt.Sprintf("IPv4 local broadcast discovery on port %d", port)
```

Comment:
Felt a bit weird at first to have such a nice human readable string as an identifier, then again it's actually nice :)  
I'd shorten it to `ipv4Identity` and so on.

Reply:
So there are 2 identities now, one is printed in the logs on startup, which is verbose, another one which is shown in the UI for errors (which we don't want to be long).

The one in the UI just says "IPv4 local" which doesn't distinguish the port, so is of no use to me for this purpose.
I guess I could change that, but I can't think of a way of keeping it short, yet still human readable, hence why we have 2 now.

The global discovery server names in the UI is just `global@http:/blabla` I guess we could have `local-ipv4@:21027`, but these things are not very friendly for the logs.

Reply:
I missed that it was also used for logging, thought it was just a map key. Anyway sounds all good - I agree exposing more info like ports is not that valuable and potentially confusing (at the very least more complex).